### PR TITLE
nodecellar vca

### PR DESCRIPTION
**Pull request by pacogomez**

### CODE REVIEW

**Comment by pacogomez** [comment]
This is a nodecellar blueprint tested on vCA Blueprinting with vCA (OnDemand) account
It is based on 
https://github.com/vmware/tosca-blueprints/blob/master/regresion-blueprints/nodecellar-token-auth.3.2.1.m3.yaml

File: nodecellar/scripts/install_postgresql.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```
Comment (denismakogon) [inline nodecellar/scripts/install_postgresql.sh]: why do you need postgresql ?

File: nodecellar/scripts/uninstall_postgresql.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (denismakogon) [inline nodecellar/scripts/uninstall_postgresql.sh]: same here, why do you need this ?

File: nodecellar/types/agentless.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        interfaces:
+            cloudify.interfaces.lifecycle:
+                start:
+                    implementation: fabric.fabric_plugin.tasks.run_script
+                    inputs:
+                        script_path:
+                            default: scripts/install_mongo.sh
+
+    # template for postgresql node
+    agentless.nodes.PostgreSqlDatabase:
```
Comment (denismakogon) [inline nodecellar/types/agentless.yaml]: We're talking about Nodecellar (nodejs and mongo) why do we need postgresql?

**Pull request merged**